packages/app-mobile/components/buttons/FloatingActionButton.tsx

```diff
@@ -56,7 +56,7 @@ const FloatingActionButton = (props: ActionButtonProps) => {
 		if (open) onMenuToggled();
 	}, [open, onMenuToggled]);
 
-	const mainButtonRef = useRef<View>();
+	const mainButtonRef = useRef<View>(null);
```

Comment:
After upgrading to React 19, `useRef` [requires a default value](https://react.dev/blog/2024/04/25/react-19-upgrade-guide#useref-requires-argument).